source/B2BApi/HostFactory.cs

```diff
@@ -88,7 +90,15 @@ public static IHost CreateHost(TokenValidationParameters tokenValidationParamete
                         .AddOutgoingMessagesModule(context.Configuration)
                         .AddProcessModule(context.Configuration)
                         .AddMasterDataModule(context.Configuration)
-                        .AddDataAccessUnitOfWorkModule(context.Configuration);
+                        .AddDataAccessUnitOfWorkModule()
+
+                        // Audit log (outbox publisher)
```

Comment:
```suggestion
                        // Audit log (outbox publisher)
```

This comment seems redundant to me.

Reply:
I can remove the comments, I just followed how the rest of the dependency injection setup is written.
![image](https://github.com/user-attachments/assets/27895b92-9e9c-485b-8d02-acc5768fcd94)